.github/workflows/tests.yml

```diff
@@ -309,21 +315,18 @@ jobs:
           pip install -U wheel
           pip install -U coverage
           pip install -U  "`ls dist/zope.index-*.whl`[docs]"
-          # Until repoze.sphinx.autointerface supports Sphinx 4.x we cannot use it:
-          pip install -U "Sphinx < 4"
       - name: Build docs
         env:
           ZOPE_INTERFACE_STRICT_IRO: 1
         run: |
           sphinx-build -b html -d docs/_build/doctrees docs docs/_build/html
-          sphinx-build -b doctest -d docs/_build/doctrees docs docs/_build/doctest
```

Comment:
Hmmmm.... ?  Is this because sphinxy doctests fail, or because they don't exist, or because they're already being run by another job in the matrix?

Reply:
In `.meta.toml` the value of `with-sphinx-doctests` is set to `false`.
I am going to fix this.